src/components/timeline/TrackContextMenu.js

```diff
@@ -893,6 +887,9 @@ class TimelineTrackContextMenuImpl extends PureComponent<
         {searchFilter
           ? this.renderShowProvidedTracks()
           : this.renderShowAllTracks()}
+        {rightClickedTrack === null ? (
+          <div className="react-contextmenu-separator" />
+        ) : null}
```

Comment:
I also moved this here so we can share a single separator with the next "hide" item.